package.json

```diff
@@ -90,33 +82,28 @@
     "@types/minimatch": "^3.0.0",
     "@types/node": "^8.0.24",
     "@types/shelljs": "^0.7.4",
-    "babel-eslint": "^8.2.1",
     "del-cli": "^1.1.0",
-    "eslint": "^4.18.0",
-    "eslint-config-airbnb": "^16.1.0",
-    "eslint-config-prettier": "^2.9.0",
-    "eslint-plugin-import": "^2.8.0",
-    "eslint-plugin-jsx-a11y": "^6.0.3",
-    "eslint-plugin-react": "^7.6.1",
+    "eslint": "^4.18.2",
     "husky": "^0.15.0-rc.8",
     "lint-staged": "^7.0.0",
     "npm-run-all": "^4.0.2",
-    "prettier": "^1.11.1",
-    "rxjs": "^5.0.0",
-    "rxjs-tslint-rules": "^3.14.0",
-    "stylelint": "^8.0.0",
-    "stylelint-config-standard": "^17.0.0",
-    "stylelint-no-unsupported-browser-features": "^1.0.0",
+    "prettier": "^1.10.2",
     "tslint": "^5.9.1",
-    "tslint-config-prettier": "^1.8.0",
-    "tslint-eslint-rules": "^5.0.0",
-    "tslint-microsoft-contrib": "^5.0.3",
-    "tslint-react": "^3.5.1",
     "typescript": "^2.8.1",
-    "typescript-eslint-parser": "^14.0.0",
```

Comment:
All the ESLint and TSLint plugins and extensions are also installed by `@hollowverse/config` since that's where they are actually required.